.circleci/config.yml

```diff
@@ -70,9 +70,9 @@ jobs:
           name: Ensure tablegen documentation is up-to-date
           command: |
             cd onnx-mlir/build
-            cmake --build . --target onnx-mlir-doc
+            cmake --build . --target onnx-mlir-docs
```

Comment:
Can we just remove the circleci since it's obsolete?

Reply:
will do in a separate PR, we did disable it recently.